src/navigators/types.tsx

```diff
@@ -15,8 +15,15 @@ export type RootStackParamList = LoggedInRootParamList;
 export type RootStackScreenProps<T extends keyof RootStackParamList> =
   StackScreenProps<RootStackParamList, T>;
 
+export type LoggedInScreenProps<T extends keyof LoggedInRootParamList> =
+  CompositeScreenProps<
+    StackScreenProps<LoggedInRootParamList, T>,
+    RootStackScreenProps<keyof RootStackParamList>
+  >;
+
 export type LoggedInRootParamList = {
   app: NavigatorScreenParams<TabParamList> | undefined;
+  'Home/Circle/Thread': { post: Post; createNewComment?: boolean };
```

Comment:
Is this so we can launch a thread from a notification? Should we change the id to just be `Circle/Thread`?

Reply:
The intention was to take this out of the tab navigator so it uses the full screen. I will remove the `Home` part in the name.